CHANGELOG.md

```diff
@@ -1,8 +1,22 @@
 # Mobile Center SDK for iOS Change Log
 
+## Version 0.11.1
+
+This version contains bug fixes and improvement that changes the current behavior.
```

Comment:
I think we should mention the bugfix for Xamarin and session correlation in Crashes.